helm/sematic-server/templates/ingress.yaml

```diff
@@ -34,6 +38,17 @@ spec:
                 name: {{ $serviceName }}
                 port:
                   number: {{ $servicePort }}
+          {{- if $.Values.deployment.socket_io.dedicated }}
+          - path: {{ printf "%s/socket.io" .path | replace "//" "/" }}
```

Comment:
Why would this deduplication be needed?

Reply:
it was a hack to remove the trailing slash if someone put `foo.bar/` as the path, but i fixed it properly now.